corehq/apps/export/views.py

```diff
@@ -2128,9 +2128,7 @@ def get(self, request, *args, **kwargs):
 
             except ResourceNotFound:
                 raise Http404()
-            except Exception as e:
-                _soft_assert = soft_assert('{}@{}'.format('brudolph', 'dimagi.com'))
-                _soft_assert(False, 'Failed to convert export {}. {}'.format(self.export_id, e))
```

Comment:
this code isn't really used nearly as often as it once was, so feels fine to remove